README.md

```diff
@@ -94,3 +94,24 @@ If you do not want the animation, simply set the `speed` attribute to `false`.
 ```
 Now the percent circle will jump to the provided percent without incrementing.
 
+### Raise % Cap
+
+The directive takes an optional attribute called `cap`. The value is boolean indicating if over 100% label is allowed. Default is true (capped at 100%).
```

Comment:
You say here that the default of `cap` is `true`, but you never set the default to true. The code you wrote essentially defaults to `false` because it is just left as `undefined`.